sntools/genevts.py

```diff
@@ -60,7 +62,14 @@ def main():
             for (i, evt) in enumerate(events):
                 evt.vertex = args.detector.generate_random_vertex()
                 outfile.write(evt.ratpac_string(i, events))
-
+        if args.mcformat == 'ROOT':
```

Comment:
If I understand correctly, the exact format of this .root file is specific to the JUNO detector simulation, right? If so, it would be better to rename it to avoid potential confusion if another experiment uses a .root with a different format later.

Reply:
Right now it need an intermediate step to be used in JUNO, and just contains a TTree with the needed information for JUNO input later. To this TTree, new branches with other variables can be added if other experiments require it.

Reply:
If another experiment needs a slightly different .root file later, I’d prefer to give that format a new name. So yes, let’s rename it to `JUNO-ROOT` (or simply `JUNO`?) instead.
(And then later, if uproot or the JUNO simulation software is updated and you don’t need that intermediate step anymore, you can just go ahead and modify this output format without having to worry about compatibility with other experiments.)